src/main/java/baritone/command/defaults/ToMapCommand.java

```diff
@@ -0,0 +1,223 @@
+/*
+ * This file is part of Baritone.
+ *
+ * Baritone is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * Baritone is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with Baritone.  If not, see <https://www.gnu.org/licenses/>.
+ */
+
+package baritone.command.defaults;
+
+import baritone.api.IBaritone;
+import baritone.api.cache.ICachedRegion;
+import baritone.api.command.Command;
+import baritone.api.command.argument.IArgConsumer;
+import baritone.api.command.exception.CommandException;
+import net.minecraft.block.AirBlock;
+import net.minecraft.block.BlockState;
+
+import javax.imageio.ImageIO;
+import java.awt.*;
+import java.awt.image.BufferedImage;
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.LinkedHashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+import java.util.stream.Stream;
+
+public class ToMapCommand extends Command {
+
+    private HashMap<String, Color> cachedColors;
+
+    public ToMapCommand(IBaritone baritone, String... names) {
+        super(baritone, "tomap");
+        cachedColors = new HashMap<>();
+    }
+
+    @Override
+    public void execute(String label, IArgConsumer args) throws CommandException {
+        ctx.worldData().getCachedWorld().reloadAllFromDisk();
```

Comment:
Shouldn't it as well repack everything it can repack so the data is more up to date?

Reply:
I don't know what repack means

Reply:
It takes the chunks in render distance and caches them, so basically whenever you use the cache and have enough time, do a repack so you don't let the cache fool you